personalize.admin.inc

```diff
@@ -966,16 +1149,11 @@ function personalize_agent_form_ajax_submit($form, $form_state) {
   // Update the current agent status.
   if (!empty($agent_data)) {
     $current_status = personalize_agent_get_status($agent_data->machine_name);
-    list($next_status, $status_text) = _personalize_status_toggle_next($current_status);
     $settings = array(
       'personalize' => array(
         'campaigns' => array(
           $agent_data->machine_name => array(
             'status' => $current_status,
-            'nextStatus' => array(
```

Comment:
This will prevent the unibar from updating properly when a campaign changes status via the ajax form.  Is there some way that acquia lift knows to update this information in response to this?